src/posts/2020-09-29-introducing-WAI.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Introduction to Web Accessibility Initiative.
+description: Key facts about web accessibility initiative, what it is, and why it exists.
+category: Background
+author: Hamsa Harcourt
+further_reading:
+  - title: About W3C WAI
+    url: https://www.w3.org/WAI/about/
+    source: W3C Web Accessibility Initiative
+  - title: "What is Web Accessibility Initiative (WAI)?"
+    url: https://www.washington.edu/doit/what-web-accessibility-initiative-wai
+    source: "DO-IT - Disabilities, Opportunities, Internetworking, and Technology"
+  - title: Web Accessibility Initiative
+    url: https://en.wikipedia.org/wiki/Web_Accessibility_Initiative
+    source: Wikipedia
+date: 2020-09-29
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - background
+---
+
+The advent of the web brought infinite possibilities. It changed the way we learn, work and socialize because we were no longer limited by borders; real or imagined. However there was a problem: Lack of standard guidelines and proper structure. Due to the lack of standard guidelines and the divergence of individual initiatives that provided no real solution to people, the **Web Accessibility Initiative**(WAI) was created.

```

Comment:
```suggestion
The advent of the web brought infinite possibilities. It changed the way we learn, work and socialize because we were no longer limited by borders; real or imagined. However there was a problem: Lack of standard guidelines and proper structure. Due to the divergence of individual initiatives the [Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/) was created.
```

Removing "no real solution" as many of those initiatives helped shape the WAI, either directly or indirectly.

Additionally, linking to the WAI homepage in context may be helpful here.